### ISSUE TITLE
Undocumented functionality, adding extra fields

### ISSUE THREAD

**Issue by mmunz** [Klickfabrik/kf_backend_mod#2]
Hi,

thanks for the EXT, i came mostly for the Save&Close Buttons and did some quick testing.

I'm wondering if adding adding some extra fields in https://github.com/Klickfabrik/kf_backend_mod/blob/main/ext_tables.php#L31 just slipped in? If not, it should probably be documented.

**Comment by Klickfabrik** [comment]
Yes you are right. I will document the two fields. First is a class field for page to handle more with css and seo title if you want a other title for the current page.

thanks for asking and remember me to add this in the documentation. Marc
